web_src/css/repo.css

```diff
@@ -2282,3 +2282,110 @@ tbody.commit-list {
 .branch-selector-dropdown .scrolling.menu .loading-indicator {
   height: 4em;
 }
+
+.commits_list_group_by_date.timeline {
```

Comment:
I do not remember there are underline style naming for CSS class selectors in this code base.